.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
```

Comment:
What is the use of this line?

Reply:
GitHub displays the names of your workflows on your repository's actions page.

Reply:
Wrong - what do you mean by `push` (`on` -> `push`)here?